contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -17,11 +17,11 @@ GRANT SELECT on sys.babelfish_sysdatabases TO PUBLIC;
 -- BABELFISH_SCHEMA_PERMISSIONS
 CREATE TABLE sys.babelfish_schema_permissions (
   dbid smallint NOT NULL,
-  schema_name NAME NOT NULL,
-  object_name NAME NOT NULL,
-  permission NAME NOT NULL,
-  grantee NAME NOT NULL,
-  object_type NAME,
+  schema_name NAME NOT NULL COLLATE sys.database_default,
+  object_name NAME NOT NULL COLLATE sys.database_default,
+  permission NAME NOT NULL COLLATE sys.database_default,
+  grantee NAME NOT NULL COLLATE sys.database_default,
+  object_type NAME COLLATE sys.database_default,
```

Comment:
We have still not addressed the MOST IMPORTANT issue with the Grant on schema code - Memory concerns on peermission/grantee being individual entries

Reply:
Yes, as discussed last week we will handling that in a separate PR. 
Please read the description to check the issues handled in this PR.